pkg/urbit/vere/newt.c

```diff
@@ -219,6 +218,9 @@ _newt_consume(u3_moat* mot_u)
       if ( 8ULL <= mot_u->len_d ) {
         _newt_gain_mess(mot_u);
       }
+      else {
+        break;
+      }
```

Comment:
This fixes the bug introduced in #2042. If we have "stray bytes", but not enough to extract a message length, we must break out of the loop.